cli-commands/compiler/etherlime-artifactor/index.js

```diff
@@ -30,42 +33,64 @@ class Artifactor {
     output_path = `${output_path}.json`;
 
     let finalObject = object;
-    
+
     try {
       let json = await fs.readFile(output_path, { encoding: "utf8" });
 
       let existingObjDirty = JSON.parse(json);
 
-      finalObject = Schema.normalize(existingObjDirty);
+      if (!skipNormalize) {
+        finalObject = Schema.normalize(existingObjDirty);
+      }
       let finalNetworks = {};
 
       _.merge(finalNetworks, finalObject.networks, object.networks);
       _.assign(finalObject, object);
 
       finalObject.networks = finalNetworks;
-    
+
     } catch (error) {
-       
+
     }
 
     finalObject.updatedAt = new Date().toISOString();
     await fs.outputFile(output_path, JSON.stringify(finalObject, null, 2), "utf8");
+    if (extra_opts && extra_opts.abiOnly) {
```

Comment:
Could we use the other approach here, where we use NOT operator (!) and return the appropriate func for execution?